component/service/src/main/java/org/exoplatform/social/rest/impl/userrelationship/UsersRelationshipsRestResourcesV1.java

```
@@ -142,10 +142,18 @@ public Response createUsersRelationships(@Context UriInfo uriInfo,
     if (sender == null || receiver == null) {
       throw new WebApplicationException(Response.Status.UNAUTHORIZED);
     }
+
+    Relationship.Type status = null;
+    if (model.getStatus() != null) {
+      try {
+        status = Relationship.Type.valueOf(model.getStatus().toUpperCase());
+      } catch (Exception e) {
+        // Ignore invalid status.
```

Ok good, actually it is never good to have an empty catch block.